File: dieta-app SPRING/src/main/java/com/dietaapp/modelo/HistoriaSalud.java
package com.dietaapp.modelo;

import jakarta.persistence.Entity;
import jakarta.persistence.GeneratedValue;
import jakarta.persistence.GenerationType;
import jakarta.persistence.Id;
import lombok.AllArgsConstructor;
import lombok.Data;
import lombok.NoArgsConstructor;
import lombok.ToString;

import java.util.ArrayList;

@Entity
@Data
@NoArgsConstructor
@AllArgsConstructor
@ToString
public class HistoriaSalud {
  @Id
  @GeneratedValue(strategy = GenerationType.IDENTITY)
  int idHistoriaSalud;
  int idPaciente;
  ArrayList<String> enfermedades;
  double imc;
  int estatura;
}

File: app-dieta/src/app/clases/HistoriaSalud.ts
export class HistoriaSalud{
    idHistoria : number;
    idPaciente : number;
    enfermedades : string;    
    imc :number;
    estatura : number;
    peso : number;

    constructor(idPaciente : number,
        enfermedades : string,
        imc : number,
        estatura : number,
        peso : number
        ){
            this.idPaciente = idPaciente;
            this.enfermedades = enfermedades;
            this.imc = imc;
            this.estatura = estatura;
            this.peso = peso;

        }

}